src/plugins/credential_management/public/components/breadcrumbs.ts

```diff
@@ -22,19 +22,19 @@ export function getCreateBreadcrumbs() {
     ...getListBreadcrumbs(),
     {
       text: i18n.translate('credentialManagement.credentials.createBreadcrumb', {
-        defaultMessage: 'Create credentials',
+        defaultMessage: 'Create Stored Credential',
       }),
       href: `/create`,
     },
   ];
 }
 
-export function getEditBreadcrumbs(credentials: CredentialEditPageItem) {
+export function getEditBreadcrumbs(credential: CredentialEditPageItem) {
```

Comment:
why updating credential management UI? let's do a separate PR for management UI changes

Reply:
That's not my change. Upstream had a force push that leads to some diffs. I'll fix it